agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -325,17 +314,46 @@ class Feedback(BaseModel):
 
 class TraceDB(Document):
     app_id: Optional[str]
-    variant_id: str
-    spans: List[PydanticObjectId]
+    base_id: str
+    config_name: str
     start_time: datetime
     end_time: datetime = Field(default=datetime.now())
     cost: Optional[float]
-    latency: float
+    environment: Optional[str]
+    latency: Optional[float]
     status: str  # initiated, completed, stopped, cancelled, failed
+    type: str = Field(default="generation")
     token_consumption: Optional[int]
     user: Link[UserDB]
     tags: Optional[List[str]]
     feedbacks: Optional[List[Feedback]]
+    created_at: datetime = Field(default=datetime.now())
 
     class Settings:
         name = "traces"
+
+
+class SpanDB(Document):
+    trace: Link[TraceDB]
+    parent_span_id: Optional[str]
+    meta: Optional[Dict[str, Any]]
+    event_name: str  # Function or execution name
+    event_type: Optional[str]
+    start_time: datetime
+    duration: Optional[int]
+    status: SpanStatus
+    environment: Optional[str]
+    end_time: datetime = Field(default=datetime.now())
+    inputs: Optional[Dict[str, Any]]
+    outputs: Optional[List[str]]
+    prompt_user: Optional[str]
```

Comment:
> please remove. In case of an llm call we would save the input to the llm call in inputs (as a dict:
> 
> {"prompt": [{"system":"xx"},{"fsad": "asdf"}}]}

In the Observability document that you shared, I added the new changes for the proposed Trace and Spans document. Could you please check it out?

Reply:
Thanks! Added the comments there